flink-python/pyflink/table/tests/test_expression.py

```diff
@@ -299,6 +299,10 @@ def test_expressions(self):
                              lit(2).hours)))
         self.assertEqual("dateFormat(time, '%Y, %d %M')",
                          str(date_format(col("time"), "%Y, %d %M")))
+        self.assertEqual("dateFormat(toTimestamp('1970-01-01 08:01:40'), 'yyyy-MM-dd HH:mm:ss')",
+                         str(date_format(to_timestamp('1970-01-01 08:01:40'), "yyyy-MM-dd HH:mm:ss")))
+        self.assertEqual("dateFormat('1970-01-01 08:01:40', 'yyyy-MM-dd HH:mm:ss')",
```

Comment:
could you add a test with the format that the DateTimeFormatter could cope with - with precision and timezone

Reply:
Hi @davidradl , sure I can add that. 
Also want to point out that these test_expressions don't test any implementations, they just test the "signature" of defined functions.